.github/workflows/ci.yml

```diff
@@ -26,9 +26,10 @@ jobs:
       - name: Checkout sail on api
         uses: actions/checkout@v2
         with:
-          repository: darpa-sail-on/sail-on-api
+          repository: darpa-sail-on/Sail-On-API
```

Comment:
General note: AFAIK, these URLs are not case sensitive. I routinely clone repositories using all lowercase, because I find anything else to be irritating by default.

Reply:
Ah didn't realize that, I'll update it in a separate PR that would address https://github.com/darpa-sail-on/sail-on-client/issues/82

Reply:
@waxlamp would it be okay if I merge this PR?

Reply:
Yes, that's what I meant when I said to go with Najam's review.